File: app/src/main/java/com/inz/z_inz/activity/MyApplication.java
package com.inz.z_inz.activity;

import android.app.Application;
import android.content.Context;

import com.inz.z_inz.util.FileUtils;
import com.inz.z_inz.util.SPHelper;

import java.io.File;

/**
 * Create By 11654
 *
 * @author Zhenglj
 * @version 1.0.0
 * @date Create By 2018/7/21 10:13
 */
public class MyApplication extends Application {

    public static boolean escFlag = false;
    // 上下文
    private Context mContext;
//    private


    public MyApplication() {
        super();
    }

    @Override
    public void onCreate() {
        super.onCreate();

        mContext = this;
        // 初始化 SharePreferences
        SPHelper.getInstance().initSharedPreferences(mContext);

        createFile();
    }

    @Override
    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
    }

    @Override
    public void onTerminate() {
        super.onTerminate();
    }

    /**
     * 创建文件
     */
    private void createFile() {
        File file = new File(FileUtils.getSDPath() + "/com.inz.z_inz/");
        if (!file.exists()) {
            file.mkdir();
        }
    }

}

File: zlauncher/src/main/java/com/inz/z/zlauncher/MyApplication.kt
package com.inz.z.zlauncher

import android.app.Application
import com.inz.z.base.util.L

/**
 *
 * @author Zhenglj
 * @version 1.0.0
 * Create by inz in 2019/12/31 10:20.
 */
class MyApplication : Application() {
    companion object {
        const val TAG = "MyApplication"
    }

    override fun onCreate() {
        super.onCreate()
        L.initL(applicationContext)

    }
}